standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/Microsoft.VisualBasic.Core.xml

```diff
@@ -4,201 +4,6 @@
     <name>Microsoft.VisualBasic.Core</name>
   </assembly>
   <members>
-    <member name="T:Microsoft.VisualBasic.ApplicationServices.ApplicationBase">
```

Comment:
Looks like these changes are expected:
https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/api-diff/preview9/.Net/3.0-preview9_Microsoft.VisualBasic.ApplicationServices.md

Reply:
Indeed. I see those going away when I reflected for Preview 9.